src/Microsoft.ML.StandardTrainers/Standard/Online/AveragedPerceptron.cs

```diff
@@ -76,6 +77,11 @@ public sealed class AveragedPerceptronTrainer : AveragedLinearTrainer<BinaryPred
 
         private readonly Options _args;
 
+        internal class AveragedPerceptronDefault : AveragedDefault
+        {
+            public new const int NumberOfIterations = 10;
```

Comment:
In the reported issue:

![image](https://user-images.githubusercontent.com/8291187/104495495-7240ae00-559d-11eb-9564-83d9771c5d86.png)

The above API is setting

```C#
int numberOfIterations = AveragedLinearOptions.AveragedDefault.NumberOfIterations
```

adding a `new const` field here isn't going to change the API. It needs to change to reference `AveragedPerceptronDefault.NumberOfIterations`.

Reply:
I didn't realize there was another place with the old reference as well, so thanks for the catch!  I have update the extension method to point to the new class.